finality-aleph/src/testing/mocks/mod.rs

```diff
@@ -1,22 +1,28 @@
-pub(crate) use acceptance_policy::AcceptancePolicy;
-pub(crate) use backend::{create_block, Backend};
-pub(crate) use block_finalizer::MockedBlockFinalizer;
-pub(crate) use block_request::MockedBlockRequester;
-pub(crate) use justification_handler_config::JustificationRequestSchedulerImpl;
-pub(crate) use proposal::{
+pub use acceptance_policy::AcceptancePolicy;
+use aleph_primitives::BlockNumber;
+pub use backend::{create_block, Backend};
+pub use block_finalizer::MockedBlockFinalizer;
+pub use block_request::MockedBlockRequester;
+pub use client::{TestClient, TestClientBuilder, TestClientBuilderExt};
+pub use justification_handler_config::JustificationRequestSchedulerImpl;
+pub use proposal::{
     aleph_data_from_blocks, aleph_data_from_headers, unvalidated_proposal_from_headers,
 };
-pub(crate) use session_info::{SessionInfoProviderImpl, VerifierWrapper};
+pub use session_info::{SessionInfoProviderImpl, VerifierWrapper};
+use sp_runtime::traits::BlakeTwo256;
+use substrate_test_runtime::Extrinsic;
 
-pub(crate) type TBlock = substrate_test_runtime::Block;
-pub(crate) type THeader = substrate_test_runtime::Header;
-pub(crate) type THash = substrate_test_runtime::Hash;
-pub(crate) type TNumber = substrate_test_runtime::BlockNumber;
+type Hashing = BlakeTwo256;
+pub type TBlockNumber = BlockNumber;
```

Comment:
Hm, since the whole purpose of this exercise is to make `BlockNumber` the only block number we use I think we should just use it for tests as well, directly and without this alias. What do you think?

Reply:
Yeah, I thought about it. We can change all those occurrences to `BlockNumber`. My thinking was to leave it as it is in case it changes in the future but it will likely never happen.

Reply:
Yeah, I would change it to `BlockNumber` then.